cmd/preview/deploy.go

```diff
@@ -198,7 +196,7 @@ func (pw *Command) waitUntilRunning(ctx context.Context, name, namespace string,
 	return nil
 }
 func (pw *Command) waitToBeDeployed(ctx context.Context, name string, a *types.Action, timeout time.Duration) error {
-	return pw.okClient.Pipeline().WaitForActionToFinish(ctx, name, okteto.Context().Namespace, a.Name, timeout)
+	return pw.okClient.Pipeline().WaitForActionToFinish(ctx, name, name, a.Name, timeout)
```

Comment:
Is this correct? Does the name contains the namespace to be used to query the API?

Reply:
Yes, If we use the namespace stored in the context is not going to be the preview env anymore...

Reply:
but the flag `--wait` won't work. I mean, that namespace is being used to do a query to the API

Reply:
name is the preview namespace, it should work as before